INSTALL.txt

```diff
@@ -68,19 +68,21 @@ Windows
 Essential modules
     cheetah-2.0.1+          use "pip install cheetah"
     par2cmdline >= 0.4      http://parchive.sourceforge.net/
+                            Beware effect of -N parameter introduced in v0.6.13
```

Comment:
In the current release (v0.6.13) -N will include missing blocks, but in upcoming release the par2cmdline guy has inverted the behavior and -N will exclude missing blocks. Very confusing..